src/app.js

```diff
@@ -37,54 +37,23 @@ export const weather = (appId, store, wuser, wpassword, token) =>
     // be sent asynchronously
     res.status(201).end();
 
-    // Handle messages identified as action requests
-    events.onAction(req.body, appId, token,
-      (action, focus, message, user) => {
+    // Handle messages identified as intent requests
```

Comment:
I'd suggest to rephrase as something like 'messages on which an intent has been recognized'